KuringLite/Search/View/SearchedRecentView.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  SearchedRecentList.swift
+//  KuringLite
+//
+//  Created by Hamlit Jason on 2022/06/07.
+//
+
+import SwiftUI
+import KuringCommons
+
+struct SearchedRecentList: View {
+    @EnvironmentObject private var engine: SearchEngine
```

Comment:
```suggestion
    @EnvironmentObject var engine: SearchEngine
```
통일성을 위해 `private` 을 제거합니다.

Reply:
제거